saleor/graphql/file_upload/views.py

```diff
@@ -1,58 +1,177 @@
         else:
-            response = super().dispatch(request, *args, **kwargs)
+            response = self.handle_query(request, *args, **kwargs)
         # Add access control headers
-        response['Access-Control-Allow-Origin'] = ','.join(
-            settings.ALLOWED_HOSTS)
+        response['Access-Control-Allow-Origin'] = (
+            settings.ALLOWED_GRAPHQL_ORIGINS)
         response['Access-Control-Allow-Methods'] = 'POST, OPTIONS'
         response['Access-Control-Allow-Headers'] = (
             'Origin, Content-Type, Accept, Authorization')
         return response
 
-    @staticmethod
-    def get_graphql_params(request, data):
-        content_type = GraphQLView.get_content_type(request)
-
-        # Only check multipart/form-data if content_type is not None (this is
-        # not checked in the original version from `graphene-file-upload`).
-        if content_type and 'multipart/form-data' in content_type:
-            query, variables, operation_name, id = super(
-                FileUploadGraphQLView,
-                FileUploadGraphQLView).get_graphql_params(request, data)
-            operations = data.get('operations')
-            files_map = data.get('map')
-            try:
-                operations = json.loads(operations)
-                files_map = json.loads(files_map)
-                variables = operations.get('variables')
-                for file_key in files_map:
-                    # file key is which file it is in the form-data
-                    file_instances = files_map[file_key]
-                    for file_instance in file_instances:
-                        test = obj_set(
-                            operations, file_instance, file_key, False)
-                query = operations.get('query')
-                variables = operations.get('variables')
-            except Exception as e:
-                raise e
+    def handle_query(self, request: HttpRequest, *args, **kwargs):
+        if request.method == 'GET':
+            return render_to_response('graphql/playground.html')
+        if request.method != 'POST':
+            return HttpResponseNotAllowed(
+                ['POST'],
+                content_type='application/json',
+                content=json.dumps({
+                    'errors': [
+                        self.format_error(
+                            'GraphQL only supports POST requests.')]}))
+        try:
+            data = self.parse_body(request)
+        except ValueError as e:
+            return HttpResponseBadRequest(
+                content_type='application/json',
+                content=json.dumps({
+                    'errors': [self.format_error('Unable to parse query.')]}))
+        if isinstance(data, list):
+            responses = [self.get_response(request, entry) for entry in data]
+            result = [response for response, code in responses]
+            status_code = max((code for response, code in responses),
```

Comment:
Shouldn't you add a blank line after `max(`?

Reply:
That's the output of YAPF you're looking at 😄